Ginger/GingerCoreNET/Run/RunListenerLib/ExecutionLoggerManager.cs

```diff
@@ -322,21 +322,27 @@ public override void BusinessFlowStart(uint eventTime, BusinessFlow businessFlow
         public override void BusinessFlowEnd(uint eventTime, BusinessFlow businessFlow, bool offlineMode = false)
         {
             mContext.BusinessFlow = businessFlow;
-            Object BFR = mExecutionLogger.SetReportBusinessFlow(mContext, offlineMode, ExecutedFrom, this.Configuration.ExecutionLoggerConfigurationIsEnabled);
+            Object BFR = null;
 
-            if (this.Configuration.ExecutionLoggerConfigurationIsEnabled)
+            try
             {
-                if (this.ExecutedFrom == Amdocs.Ginger.Common.eExecutedFrom.Automation)
+                BFR = mExecutionLogger.SetReportBusinessFlow(mContext, offlineMode, ExecutedFrom, this.Configuration.ExecutionLoggerConfigurationIsEnabled);
+                if (this.Configuration.ExecutionLoggerConfigurationIsEnabled)
                 {
-                    this.ExecutionLogBusinessFlowsCounter = 0;
-                    mExecutionLogger.ExecutionLogBusinessFlowsCounter = 0;
-                    this.BFCounter = 0;
+                    if (this.ExecutedFrom == Amdocs.Ginger.Common.eExecutedFrom.Automation)
```

Comment:
did you tested the changes also for regular executions on Automate and Run set to make sure Report comes as expected?